lib/bump/version_control.rb

```diff
@@ -0,0 +1,44 @@
+module Bump
+  class VersionControlNotFoundError < StandardError; end
+
+  class VersionControl
+    def self.git?
```

Comment:
I try to when in doubt make things private, if we later change/move that then nobody will be surprised that these methods went away